requirements.txt

```diff
@@ -5,12 +5,14 @@ click==6.6
 cryptography==1.4
 fauxfactory==2.0.9
 Inflector==2.0.11
+import_string==0.1.0
```

Comment:
@rochacbruno, we are using this library here in robotello by import,  think it should be as a dependency. if it's installed by other packages that's ok, this way other packages are free to change their dependencies.